tests/rptest/utils/si_utils.py

```diff
@@ -242,6 +242,39 @@ def get_on_disk_size_per_ntp(chk):
     return size_bytes_per_ntp
 
 
+def get_expected_ntp_restored_size(chk, retention_policy):
+    """ Get expected retestored ntp disk size
+    We expect that redpanda will restore max
+    amount of segments with total size less
+    than retention_policy
+    """
+    size_bytes_per_ntp = {}
+    segments_sizes_per_ntp = {}
+    for _, data in chk.items():
```

Comment:
This is hard to follow, names like `data` and `chk` are quite obscure

Reply:
As well as nicer names, you could declare types in the function definition `(chk: SometType, retention_policy: SomeType)` to help the reader understand